workers/unity/Packages/io.improbable.gdk.testutils/ActionMeasurement.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using Unity.PerformanceTesting;
+
+namespace Improbable.Gdk.TestUtils
+{
+    public class ActionMeasurement
+    {
+        private readonly Action action;
+
+        private readonly List<SampleGroup> sampleGroups = new List<SampleGroup>();
+
+        private int warmupCount = 5;
+        private int measurementCount = 10;
+        private int iterationsPerMeasurement = 1;
+
+        private ActionMeasurement(Action action)
+        {
+            this.action = action;
+        }
+
+        public static ActionMeasurement Measure(Action action)
+        {
+            return new ActionMeasurement(action);
+        }
```

Comment:
What's the reason for the `public static` method that just calls and returns the private constructor? Just for the fluent API?

Reply:
Yup, wanted this to be similar to the existing performance testing API